docs/DeveloperGuide.adoc

```diff
@@ -512,12 +512,12 @@ Tags that do not exist inside the specified `**Transaction**` will be ignored, a
 
 Again, since the user input is valid, a `**UntagCommand**` command object is created and executed. As a result, a copy of the `**Transaction**` object with the specified `**Tag**` deleted replaces the original in the `**TransactionList**` at the same position.
 
-image::tag-untag/TagUntag1.png[]
+image::tag-untag/TagUntag1.png[,450]
 
-The following activity diagrams summarises what happens when the user executes a command to **tag** a `**Transaction**`.
+The following activity diagrams summarises what happens when the user executes a command to tag a `**Transaction**`.
```

Comment:
The following activity diagrams **summarise** what happens...

Reply:
Fixed.